client/packages/common/src/ui/layout/tables/material-react-table/components/RequiredNumberInputCell.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
```

Comment:
I'm not entirely clear why we need a separate component for the "Required" version of this?

Reply:
The difference is using state vs buffer state - in the usual number cell, the input state can be overwritten externally. But with one we want -> clear input field, but set draft value to 1, so calculations always have a valid pack size as a base...

There's probably other ways to handle this, I just renamed to be a bit more generic than the old `PackSizeCell`... maybe can review the logic down the track?